src/models/Popup.spec.js

```diff
@@ -38,7 +38,7 @@ const isPopup = popup => (
   expect( popup.options ).toBeInstanceOf( Object ),
   expect( popup.userCategories ).toBeInstanceOf( Object ),
   expect( popup.customStyles ).toBeInstanceOf( Object ),
-  expect( Object.keys( popup.userCategories ) ).toStrictEqual( categories )
+  expect( Object.keys( popup.userCategories ) ).toStrictEqual( CATEGORIES )
```

Comment:
```suggestion
  expect(Object.keys(popup.userCategories)).toStrictEqual(CATEGORIES)
```

Reply:
formatting issues will be foxed when (if) we add later on pretties and eslint